Foundation/URLSession/http/HTTPBodySource.swift

```diff
@@ -40,15 +40,15 @@ internal func splitData(dispatchData data: DispatchData, atPosition position: In
 }
 
 /// A (non-blocking) source for HTTP body data.
-internal protocol _HTTPBodySource: class {
+internal protocol _BodySource: class {
```

Comment:
As this is no longer specific to HTTP, should it be moved out of the http directory?